src/util/stringToArray.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Break a string into rendered characters (graphemes), 
+ * using simpler methods of breaking strings apart doesn't take into account characters with multiple bytes.
+ * For instance `'👱🏽‍♂️'.length === 7`
+ * @param {string} string - input string
+ * @return {string[]} - the string broken down into an array of characters.
+ */
+export function stringToArray (string) {
+  return [...new Intl.Segmenter().segment(string)].map(x => x.segment);
```

Comment:
`Intl.Segmenter` segments by grapheme clusters by default. This is possibly overkill for fixing #9. In some scripts, it will group whole syllables of multiple characters. If the issue is just that surrogate pairs are getting split up, then use the spread operator (`[...string]`) or the `string[Symbol.iterator]()` iterator directly on the string you want to split, as described in [this documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#utf-16_characters_unicode_code_points_and_grapheme_clusters). The string iterator is much more widely supported than `Intl.Segmenter`, which only landed in Firefox a few months ago.